config/reports.json

```diff
@@ -6,10 +6,12 @@
 		"loading-speed",
 		"search-engine-optimization",
 		"page-weight",
-		"chrome-ux-report"
+		"chrome-ux-report",
+		"progressive-web-apps"
 	],
 	"_featured": [
-		"state-of-the-web"
+		"state-of-the-web",
+		"progressive-web-apps"
```

Comment:
Let's remove this from featured for now.

Reply:
Addressed in https://github.com/HTTPArchive/httparchive.org/pull/51/commits/62865d1b58440193f5393437ad1f678c53b7b37a.